tools/refguide_check.py

```diff
@@ -280,6 +352,23 @@ def check_items(all_dict, names, deprecated, others, module_name, dots=True):
     """
     Check that `all_dict` is consistent with the `names` in `module_name`
     For instance, that there are no deprecated or extra objects.
+
+    Parameters
+    ----------
+    all_dict : list
+
+    names : set
+
+    deprecated : list
+
+    others : list
+
+    module_name : ModuleType
+
```

Comment:
dots: Boolean, whether to print a dot for each check

Reply:
Rectified